### PR TITLE
[Suyabay][#103663860] Password Reset

### PR DESCRIPTION
**Pull request by ibonly**
Feature password reset

### CODE REVIEW

File: public/js/PasswordReset.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+  var data =
+    {
+        url        : url,
+        parameter  :
+        {
+            _token : token,
+            email  : email
+        }
+    }
```
Comment (unicodeveloper) [inline public/js/PasswordReset.js:107]: Fix HoundCI issues

File: public/js/PasswordReset.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+function ajaxCall ( data, functionName )
+{
+    $.post( data.url, data.parameter)
+
+    .done( function (response)
+    {
+        ajaxLogic( data, response, functionName );
+    })
+    .fail( function (response) {
+        console.log('this action is bad');
```
Comment (unicodeveloper) [inline public/js/PasswordReset.js]: Address this

File: public/js/PasswordReset.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+/*
+| passwordReset
+| create user information and url in to an array of object i.e @data
+| and make and ajax class to function ajaxCall() by sending @data and @functionName
+*/
+function passwordReset ()
+{
+  var url   = "/password/email";
+  var email = $('#email').val();
```
Comment (unicodeveloper) [inline public/js/PasswordReset.js:96]: Fix!

File: public/js/PasswordReset.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+/*
+| passwordResetErrorAlert
+| gives error report to user
+| receives 1 parameter
+*/
+function passwordResetErrorAlert ()
+{
+    swal("Ooops!!!", "Email address does not exit", "error")
```
Comment (unicodeveloper) [inline public/js/PasswordReset.js]: Fix the missing semicolon issue

**Comment by unicodeveloper** [comment]
Your pull request should have a title that signifies what you did here

File: public/js/PasswordReset.js (hunk 1 above)
Comment (ibonly) [inline public/js/PasswordReset.js:107]: Okay

File: public/js/PasswordReset.js (hunk 2 above)
Comment (ibonly) [inline public/js/PasswordReset.js]: Removing 'console.log' will resolve the issue

File: public/js/PasswordReset.js (hunk 3 above)
Comment (ibonly) [inline public/js/PasswordReset.js:96]: Okay

File: public/js/PasswordReset.js (hunk 4 above)
Comment (ibonly) [inline public/js/PasswordReset.js]: Okay

**Comment by ibonly** [comment]
Houndci issues has been fixed

**Comment by unicodeveloper** [comment]
rebase to fix the conflicts

**Pull request merged**